sbt-cross/src/main/scala/sbtcross/JvmCross.scala

```diff
@@ -0,0 +1,37 @@
+package sbtcross
+
+import sbt._
+
+case object JVMPlatform extends Platform {
+  def identifier: String = "jvm"
+  def enable(project: Project): Project =
+    project.settings(crossPlatform := JVMPlatform)
```

Comment:
`crossPlatform := JVMPlatform` should actually be added `in Global` automatically using `globalSettings` in a separate `AutoPlugin` that is automatically triggered all the time.

Also, it should not be added by the `enable` method itself. The `enable` method should `enablePlugins(TheRightPlugin)`, and it is the platform's `AutoPlugin`'s job to set `crossPlatform`.

Otherwise, people will not be able to use `%%%` in non-`crossProject` projects. For example, in their JS-only `client` project (which has `enablePlugins(ScalaJSPlugin)` but is not a `crossProject`)